app/assets/javascripts/admin/addon/components/admin-search.gjs

```diff
@@ -62,6 +64,14 @@ export default class AdminSearch extends Component {
     return !this.adminSearchDataSource.isLoaded || this.loading;
   }
 
+  get noResultsDescription() {
+    return htmlSafe(
```

Comment:
This is not blocking but I'm wondering why we need this to be marked as `htmlSafe`.